tools/relay/README.md

```diff
@@ -0,0 +1,66 @@
+## Common errors
```

Comment:
Is this file really to commit?
It seems more its for your own help.
However, its good that you have a readme however it might be good to make a description at the beginning of what the relay is actually doing